### ISSUE TITLE
shoes won't let me include ruby twitter gem

### ISSUE THREAD

**Issue by arjunc12** [shoes/shoes-deprecated#213]
i've been using this gem http://twitter.rubyforge.org/  in all of the ruby code i've been writing and there haven't been any issues. however, when i try to write ruby code used for a shoes app and i require the twitter gem it doesn't work. i've tried doing shoes.setup i.e. 

Shoes.setup do
  gem 'twitter'
end

but it always gets an error when it tries to install it.

**Comment by ashbb** [comment]
Are you using Shoes3 (0.r1514)? If so, I guess that there is a gem which requires rubygems over 1.3.5.
What platform are you using? If you use Windows, try out [shoes-0.r1783](http://librelist.com/browser//shoes/2012/2/26/red-shoes-for-windows-with-ruby-1-9-3p125/#a391477c45fabfa46397780d0b75aaef).

**Comment by PragTob** [comment]
Yep try building shoes from source, then it works. Otherwise you can try green shoes :-)
